doc/api-example.md

```diff
@@ -112,18 +112,20 @@ add some metadata to MinHashes.
 
 ```
 >>> from sourmash import SourmashSignature, save_signatures
+>>> from tempfile import mkdtemp
 >>> sig1 = SourmashSignature(minhashes[0], name=genomes[0][:20])
->>> with open('/tmp/genome1.sig', 'wt') as fp:
+>>> tempdir = mkdtemp(suffix = "temp")
+>>> with open(tempdir + '/genome1.sig', 'wt') as fp:
 ...   save_signatures([sig1], fp)
 
 ```
 
-Here, `/tmp/genome1.sig` is a JSON file that can now be loaded and
+Here, `{}/genome1.sig`.format(tempdir) is a JSON file that can now be loaded and
```

Comment:
This won't render, because this is markdown not python. I think just leave it as `genome1.sig` with no `/tmp`.

Reply:
I was just about to ask about this. Thanks!